test/playlist.test.js

```diff
@@ -1317,4 +1317,95 @@ QUnit.module('Playlist', function() {
       'no segments, live delay can\'t be calculated'
     );
   });
+
+  QUnit.test('playlistMatch', function(assert) {
```

Comment:
if playlistMatch returns a boolean, might be better to use `assert.true` and `assert.false`. Same for isAudioOnly.

Reply:
i'll switch to that.